contracts/PetPark.sol

```diff
@@ -3,5 +3,66 @@ pragma solidity ^0.8.0;
 
 
 contract PetPark {
+    enum AnimalType {None, Fish, Cat, Dog, Rabbit, Parrot}
+    enum Gender {Male, Female}
+    struct PetOwner {
+        address owner;
```

Comment:
Since address of pet owner is already being stored in `borrowed` mapping, do we need to store it here?

Reply:
@shubham-kanodia, thanks for the comment!
I was using the `address` field to check for the case where the `PetOwner` has not borrowed anything yet.
```petOwner.address == address(0)```
If I eliminate this field from the `PetOwner` struct, is there a way to check if `borrowed[address]` has been set or not?

Reply:
How about `age == 0`?

Reply:
Yeah, that can be done. Thanks!
Besides that, is there a way to check if a struct is `NULL`?
For example, had `age=0` been valid, then is there a way to detect if a struct is unset?

Reply:
No struct will not be null, just like uint has a default value of 0, the struct defaults to the default values of its members. If `age=0` was valid then you would have no choice but to have saved `isInitialised` boolean in struct

Reply:
Got it, thanks a lot for the clarification!